lfx-mentorship/2021/01-Spring/project_ideas.md

```diff
@@ -348,3 +348,12 @@ Qualitative analysis of user interview recordings for Jobs-to-Be-done study
 - Recommended Skills: React, CSS, Ruby, Rails, Github
 - Mentor(s): Joe Kutner (@jkutner), Travis Longoria (@elbandito)
 - Upstream Issue (URL): https://github.com/buildpacks/registry-api/issues/21
+
+#### Integrate LitmusChaos with Kyevrno
```

Comment:
@realshuting what is the primary CNCF project here - LitmusChaos or Kyverno? Please select one :)

Reply:
Updated.